pytr/account.py

```diff
@@ -52,19 +51,13 @@ def login(phone_no=None, pin=None, web=True):
             print("Please enter your TradeRepublic pin:")
             pin = getpass(prompt="Pin (Input is hidden):")
 
-        print('Save credentials? Type "y" to save credentials:')
-        save = input()
-        if save == "y":
+        if store_credentials:
             with open(CREDENTIALS_FILE, "w") as f:
                 f.writelines([phone_no + "\n", pin + "\n"])
 
             log.info(f"Saved credentials in {CREDENTIALS_FILE}")
 
-        else:
-            save_cookies = False
-            log.info("Credentials not saved")
-
-    tr = TradeRepublicApi(phone_no=phone_no, pin=pin, save_cookies=save_cookies)
+    tr = TradeRepublicApi(phone_no=phone_no, pin=pin, save_cookies=True)
```

Comment:
Same critique here, the default behavior should not be to leave clear file cookies on your system indefinitely.
```suggestion
        else:
            save_cookies = False
            log.info("Credentials not saved")

    tr = TradeRepublicApi(phone_no=phone_no, pin=pin, save_cookies=save_cookies)
```